components/reddit/sources/new-link-on-a-subreddit/new-link-on-a-subreddit.js

```diff
@@ -0,0 +1,84 @@
+const reddit = require("../../reddit.app.js");
+const get = require("lodash.get");
+
+module.exports = {
+  key: "new-link-on-a-subreddit",
+  name: "New Link on a subreddit",
+  description: "Emmits a new link submitted to a subreddit",
+  version: "0.0.2",
+  props: {
+    reddit,
+    timer: {
+      label: "Polling schedule",
+      description:
+        "Pipedream poll for new links and emit accordingly with this schedule.",
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 10, // by default, run every 10 minute
```

Comment:
Another style thing (we need to document all these somewhere :) ) - our standard default is normally 15 minutes.

Reply:
Oh, good to know. I just thought 10 minutes was good enough for polling for new emitting objects. Keeping to 15 as default.